kim_cole/Assignments/CoinTosses.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+def tosses():
+	print "Starting the program"
+	trial = 1
+	head_count = 0
+	tail_count = 0
+	output = ""
+	output_string_complete = ""
+
+	for i in range (0,5000):
+		randomToss = random.randint(0,1)
+		if randomToss == 1:
+			head_count += 1
+			output = "head"
+			print "Attempt #", trial, ": Throwing a coin... It's a ", output, "! Got ", head_count, "head(s) so far and", tail_count, "tail(s) so far"
+		else:
+			tail_count += 1
+			output = "tail"
+			print "Attempt #", trial, ": Throwing a coin... It's a ", output, "! Got ", head_count, "head(s) so far and", tail_count, "tail(s) so far"
```

Comment:
Any time you're repeating something, look to see if there's a way to condense it down.  In this case you have the exact same print statement twice.  set your variables you need inside the if/else part, and then just use the print statement once outside of the if/else with the set variables